admin/company/settings/sso-management.md

```diff
@@ -10,12 +10,15 @@ title: Manage Single Sign-On
 
 ### Remove a domain from an SSO connection
 
-1. In the **Single Sign-On Connection** table, select the **Action** icon and then **Edit connection**.
-2. Select **Next** to navigate to the section where the connected domains are listed.
-3. In the **Domain** drop-down, select the **Remove** icon next to the domain that you want to remove.
-4. Select **Next** to confirm or change the connected organization(s).
-5. Select **Next** to confirm or change the default organization and team provisioning selections.
-6. Review the **Connection Summary** and select **Save**.
+1. Sign in to [Docker Admin](https://admin.docker.com){: target="_blank" rel="noopener" class="_"}.
+2. In the left navigation, select your company in the drop-down menu.
+3. Select **SSO & SCIM**.
+4. In the table, select the **Action** icon and then **Edit connection**.
+5. Select **Next** to navigate to the section where the connected domains are listed.
+6. In the **Domain** drop-down, select the **Remove** icon next to the domain that you want to remove.
```

Comment:
Question for clarity: is the **Remove** icon the small **x**? I also see a larger **X** for the whole field that I'm assuming clears all of the domains?

Reply:
Good catch. Changed it to **x** instead of **Remove** as there's actually no tooltip with **Remove** wording.